lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -197,10 +197,10 @@ defmodule Mix.Tasks.Xref do
 
   Each item in the list is a map with the following keys:
 
-    * `:callee` - A tuple containing the module, function and arity of the call
-    * `:line` - Line where the function is called (integer)
-    * `:file` - File where the function is called (binary)
-    * `:caller_module` - Module where the function is called (atom)
+    * `:callee` - a tuple containing the module, function, and arity of the call
+    * `:line` - an integer represeting the line where the function is called
```

Comment:
Slight typo - `representing`

Reply:
Great catch, believe it or not my "n" key on my laptop has been acting up the last few days so this is far from the first time it happens 😄

Reply:
😆 That is unfortunate!